test/nl/surf/eduhub_rio_mapper/api_test.clj

```diff
@@ -82,6 +83,20 @@
      :institution-schac-home "edu.nl"}
     "/job/delete/education-specifications/12345678-1234-2345-3456-123456789abc"
 
+    {:action                 "link"
+     ::ooapi/type            "education-specification"
+     ::ooapi/id              "12345678-1234-2345-3456-123456789abc"
+     ::rio/code              "1234O4321"
+     :institution-schac-home "edu.nl"}
+    "/job/link/1234O4321/education-specifications/12345678-1234-2345-3456-123456789abc"
+
+    {:action                 "link"
+     ::ooapi/type            "education-specification"
+     ::ooapi/id              nil
+     ::rio/code              "1234O4321"
+     :institution-schac-home "edu.nl"}
+    "/job/unlink/1234O4321/education-specifications"
+
```

Comment:
Again confusing to have an "action" called "link" and an URL path starting with "/job/unlink", please add a comment.